### DISCUSSION TITLE
aria2启动 不了

### DISCUSSION THREAD

**Discussion by npcXxX** [coolsnowwolf/lede#9543] [Bugs / 问题反馈]
不论什么用户都启动不了aria2
![image](https://user-images.githubusercontent.com/43200321/173231327-30b85dca-918e-4832-8cd0-c015092e6368.png)

**Comment by EnnawYang** [comment]
![Snipaste_2022-06-12_19-39-33](https://user-images.githubusercontent.com/6705663/173231410-4401232d-250b-49fc-87ed-e0807d09762c.png)
咱可用的好好的
